ts-embed_assets/hello.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * 
+ * @klotho::execution_unit {
+ *   id = "assets-hello"
+ *   keep_warm = true
+ * }
+ */
+
+export const router = require('express').Router();
+
+router.get("/hello", async (req, res) => {
+  res.send("hi")
```

Comment:
Minor: maybe add a newline, so that you don't get the trailing `%` in the output?

```
$ curl localhost:3000/hello
hi%
  ↑
```